datapipelines/build.gradle

```diff
@@ -49,4 +49,18 @@ dependencies {
 
     testImplementation(platform(Dependencies.junitBom))
     testImplementation(Dependencies.junitJupiter)
-}
\ No newline at end of file
+
+    testRuntimeOnly(Dependencies.junitEngine)
+}
+
+tasks.withType(Test).configureEach {
```

Comment:
There is a comment in there, mock has issues with java 17
```
// https://github.com/mockk/mockk/issues/681
```

Reply:
I see the comment, but I would appreciate some explanation in our codebase so anyone seeing the task not have to go through Github issue every time to understand why it was added.

Because even after looking at the linked issue, I'm not sure why this part is added

```
useJUnitPlatform()
filter {
    excludeTestsMatching("*Integration*")
}
ignoreFailures = true
```